Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/InAppNotification.Events.cs

```diff
@@ -21,13 +21,48 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     public partial class InAppNotification
     {
         /// <summary>
+        /// Event raised when the notification is opening
+        /// </summary>
+        public event EventHandler Opening;
+
+        /// <summary>
+        /// Event raised when the notification is opened
+        /// </summary>
+        public event EventHandler Opened;
+
+        /// <summary>
+        /// Event raised when the notification is dismissing
+        /// </summary>
+        public event InAppNotificationDismissingEventHandler Dismissing;
```

Comment:
Later we may want to discuss if these should be Closing/Closed to align with FlyoutBase.

Reply:
Of course. I did not wanted to make a breaking change because the Dismissed event was already available in 2.0.

Reply:
Yup, makes sense.  Just wanted to make sure we have something tracking it for a later release.

Reply:
We need to decide on this. @nmetulev may have an opinion.

Reply:
Closed/Closing does fit in better with the Opened/Opening events. If you do want to change it, I recommend marking Dismissed deprecated and adding the new events.

Reply:
Ok.